writers/README.md

```diff
@@ -0,0 +1,16 @@
+# Writers package
+
+Writers package provides an implementation of various `message consumers`.
```

Comment:
This is not true. Message consumers can consume messages without storing them in any kind of storage. Hence, this should be replaced with `message writers`.